README.md

```diff
@@ -18,25 +32,23 @@ You should be able to access `biomage-utils` by typing:
 
     python3 biomage
 
-As a prerequisite for running all scripts in this repo, you will need a GitHub Personal Access
-Token with full access to your account. You can generate one
-[here](https://github.com/settings/tokens). Make sure you note down this and
-supply it when required. Utilities can accept this token in two ways:
 
-* either by having it available as the environment variable `GITHUB_API_TOKEN`
-* or by passing it as an option with the `-t` flag.
+### Passing credentials
+
+Tools typically require you to have sufficient AWS and GitHub credentials to function.
```

Comment:
can you expand on what you mean by `sufficient` and add a short note on how can one obtain AWS and GitHub credentials